src/index_find_regions.js

```diff
@@ -17,7 +17,8 @@ exports.handler = (event, context, callback) => {
     }
     event.awsevents.Credentials = credentials;
     credentials = new Buffer(JSON.stringify(credentials)).toString('base64');
-    event.cloudtrail.headers.Credentials = credentials;
+    var cloudtrailResult = JSON.parse(event.cloudtrail.result.body);
```

Comment:
Why are we moving cloudtrailResult from index_set_results.js to index_find_regions.js?